docs/Contributing/architecture/software/software-inventory.md

```diff
@@ -0,0 +1,8 @@
+# Software Inventory Architecture
+
+This document provides an overview of Fleet's Software Inventory architecture.
+
+## Introduction
+
+Software Inventory in Fleet provides visibility into the software installed on devices across the fleet. This document provides insights into the design decisions, system components, and interactions specific to the Software Inventory functionality.
```

Comment:
This should be a TODO. It's not wrong but it's also not informative. Likely another "write this doc" ticket is in order.

Reply:
But is it a good introduction to someone who doesn't know what software inventory is? I'm thinking from the perspective of a brand new Fleetie.

Reply:
I'll review this with Tim to determine if we want to combine these pages.

Reply:
> I'll review this with Tim to determine if we want to combine these pages.

Whoops wrong comment. 

Adding headers, this is essentially an empty doc that we want to fill later. I think we're more likely to fill it in later if the empty docs stay in place.